spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging/src/test/java/org/springframework/cloud/gcp/logging/autoconfig/StackdriverLoggingAutoConfigurationTests.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.logging.autoconfig;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.gcp.logging.TraceIdLoggingWebMvcInterceptor;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Mike Eltsufin
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = { StackdriverLoggingAutoConfiguration.class }, properties = {
+		"debug" }, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+public abstract class StackdriverLoggingAutoConfigurationTests {
+
+	@Autowired
+	private Object[] interceptors;
+
+	public abstract void test();
+
+	public int countTraceIdInterceptors() {
+		int count = 0;
+		for (int i = 0; i < this.interceptors.length; i++) {
+			if (this.interceptors[i] instanceof TraceIdLoggingWebMvcInterceptor) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	public static class StackdriverLoggingAutoConfigurationTest_default
```

Comment:
Just curious, are these tests run by Maven?
I thought the classes needed to end in "Test" or "Tests".

Reply:
Yep they run. Maven/JUnit only really care about the `@Test` annotation on the methods.

Reply:
They didn't run when I just mvn installed the starter locally.

-------------------------------------------------------
 T E S T S
-------------------------------------------------------

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

Reply:
You're right! Good catch!